### PR TITLE
[full-ci] [tests-only] Removed step and implementation of setting share auto accept

### PR DESCRIPTION
**Pull request by grgprarup**
## Description
This PR removes the step `Given auto-accept shares has been disabled` and its implementation. Auto accepting of share is disabled by default in OCIS.

## Related Issue

- Part of https://github.com/owncloud/ocis/issues/5337

## How Has This Been Tested?

- :robot:

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/coreApiMain/checksums.feature [hunk 1]
```diff
@@ -177,8 +177,7 @@ Feature: checksums
 
   @files_sharing-app-required @issue-1291
   Scenario Outline: sharing a file with checksum should return the checksum in the propfind using new DAV path
-    Given auto-accept shares has been disabled
-    And using <dav_version> DAV path
+    Given using <dav_version> DAV path
```
Comment (amrita-shrestha) [inline tests/acceptance/features/coreApiMain/checksums.feature:180]:
```suggestion
    Given using new DAV path
```

File: tests/acceptance/features/coreApiMain/checksums.feature [hunk 2]
```diff
@@ -192,8 +191,7 @@ Feature: checksums
 
   @files_sharing-app-required @issue-1291 @skipOnOcV10
   Scenario Outline: modifying a shared file should return correct checksum in the propfind using new DAV path
-    Given auto-accept shares has been disabled
-    And using <dav_version> DAV path
+    Given using <dav_version> DAV path
```
Comment (amrita-shrestha) [inline tests/acceptance/features/coreApiMain/checksums.feature:194]:
```suggestion
    Given using new DAV path
```

**Review by SagarGi — COMMENTED** [review]
LGTM :+1:

**Review by saw-jan — APPROVED** [review]
LGTM

**Pull request merged**